docs/keywords.json

```diff
@@ -4156,7 +4156,7 @@
         "category": "Generic",
         "release": "1.5"
     },
-    "wheelchair": {
+    "wheelchair-f": {
```

Comment:
do walking and wheelchair have a non-f version? Why rename these?

Reply:
they do not at the moment but we have a pattern of naming filled icons with -f

Reply:
But now any devs that used `wheelchair` in their app will get an error as you just removed it.

Reply:
Right? Maybe not?

Reply:
Thanks for bringing that up, Paul :) 
Grecia, it would be helpful if the `-f` suffix is reserved for icons that serve as an alternate filled version to a default outlined icon. In the case of `wheelchair` and `walking`, the filled shapes are just the default style for this concept necessary for legibility reasons. 

I would make the same argument for the `running` icon. What was the reason the outlined version was requested? I don't think I would suggest it become the default running icon over the filled version.

Reply:
> But now any devs that used `wheelchair` in their app will get an error as you just removed it.

Right, since there is no longer a `wheelchair` icon, their code has nothing to reference - that would be my guess as well.